modules/test/playwright/tests/account-admin-web/accountPermissions.spec.ts

```diff
@@ -696,53 +700,53 @@ test(
 
 		await page.waitForLoadState('domcontentloaded');
 
-		await setItemsPerPage(page, '4');
+		await setItemsPerPage(page, '20');
```

Comment:
why here is a string? (I know was a string also before but in the other files we are using numbers